cypress/e2e/list/list.cy.js

```diff
@@ -89,27 +89,23 @@ describe('List View', () => {
   describe('Narration buttons', () => {
     it('should filter the results to narrative-only results and back', () => {
       // Initially all is checked.
-      cy.get(addNarrativesButton).should('have.class', 'deselected');
       cy.get(removeNarrativesButton).should('have.class', 'selected');
       cy.get(filterHasNarrative).should('not.be.checked');
 
       // Click the narrative-only button.
       cy.get(addNarrativesButton).click().should('have.class', 'selected');
-      cy.get(removeNarrativesButton).should('have.class', 'deselected');
 
       cy.get(filterHasNarrative).should('be.checked');
 
       // Click the narrative-only button again. There should be no change.
       cy.get(addNarrativesButton)
         .click({ force: true })
         .should('have.class', 'selected');
-      cy.get(removeNarrativesButton).should('have.class', 'deselected');
 
       cy.get(filterHasNarrative).should('be.checked');
 
       // Click the all results button. The narratives should be removed.
       cy.get(removeNarrativesButton).click().should('have.class', 'selected');
-      cy.get(addNarrativesButton).should('have.class', 'deselected');
 
       cy.get(filterHasNarrative).should('not.be.checked');
     });
```

Comment:
Line 175 is failing this test since it can't click on something disabled. Should we just remove `.click()`?

Reply:
I'm not seeing the failure, and it's passing in CI too.

Reply:
Might be my local then. I'll go ahead and approve if you're passing and so is CI